### PR TITLE
Budjet app

### PR DESCRIPTION
**Pull request by vanelnw**
This Ruby on Rails application that allows the user to:
- register and log in, so that the data is private to them.
- introduce new transactions associated with a category.
- see the money spent on each category.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,183 @@
...
+
+### Run tests
+
+To run all tests, including unit and integration tests, run the following command:
+
+```
+  rails test:all
+```
+
+### Deployment
```
Comment (Davidosky007) [inline README.md:124]: * Kindly deploy your app as its a requirement for this capstone project [check requirement here ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=The%20project%20should%20be%20deployed%20and%20accessible%20online.)

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+.main_container {
+  width: 100%;
+  padding: 20px 0;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+
```
Comment (Davidosky007) [inline app/assets/stylesheets/application.css]: * Kindly follow the given design color as mentioned in the [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @vanelnw 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* All tests passed ✔️ 
* Good looking readme file 👍 

## Required Changes ♻️
* Good job 👏 with the video presentation but your video should contain highlight about a piece of the code you find interesting or you are proud of making. [Check requirement here](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Once%20you%20have,very%20proud%20of.)



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @vanelnw , 



In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**